CMakeLists.txt

```diff
@@ -1050,7 +1050,7 @@ endif ()
 
```

Comment:
> i#1557: compatibility update, CMake API dynamorio has changed.

grammar nit: comma isn't right there: should be a colon or sthg.  suggest sthg like: `DRi#1557: DR CMake functions now take extra arg` which gives an idea of what changed.